core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -89,95 +86,96 @@
-             class="protractor-test-creator-dashboard-link">
+             class="dropdown-item nav-link protractor-test-creator-dashboard-link">
             <span translate="I18N_TOPNAV_CREATOR_DASHBOARD"></span>
           </a>
         </li>
-        <li>
+        <li class="nav-item">
           <a ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/learner_dashboard"
-             class="protractor-test-learner-dashboard-link">
+             class="dropdown-item nav-link protractor-test-learner-dashboard-link">
             <span translate="I18N_TOPNAV_LEARNER_DASHBOARD"></span>
           </a>
         </li>
-        <li>
-          <a ng-if="($ctrl.isAdmin || $ctrl.isTopicManager)" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/topics_and_skills_dashboard"
-             class="protractor-test-topics-and-skills-dashboard-link">
+        <li class="nav-item">
+          <a ng-if="($ctrl.isAdmin || $ctrl.isTopicManager)" ng-click="$ctrl.closeSubmenuIfNotMobile($event)"
+             href="/topics_and_skills_dashboard" class="dropdown-item nav-link protractor-test-topics-and-skills-dashboard-link">
             <span translate="I18N_TOPNAV_TOPICS_AND_SKILLS_DASHBOARD"></span>
           </a>
         </li>
-        <li>
+        <li class="nav-item">
           <a ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/notifications_dashboard"
-             class="protractor-test-notifications-link">
+             class="dropdown-item nav-link protractor-test-notifications-link">
             <span translate="I18N_TOPNAV_NOTIFICATIONS"></span>
             <span ng-if="$ctrl.numUnseenNotifications > 0">
               (<[$ctrl.numUnseenNotifications]>)
             </span>
           </a>
         </li>
-        <li>
+        <li class="nav-item">
           <a ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/preferences"
-             class="protractor-test-preferences-link">
+             class="dropdown-item nav-link protractor-test-preferences-link">
             <span translate="I18N_TOPNAV_PREFERENCES"></span>
           </a>
         </li>
-        <li ng-if="$ctrl.isModerator">
-          <a ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/moderator" target="_blank" rel="noopener">
+        <li class="nav-item" ng-if="$ctrl.isModerator">
+          <a class="dropdown-item nav-link" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/moderator" target="_blank" rel="noopener">
             <span translate="I18N_TOPNAV_MODERATOR_PAGE"></span>
           </a>
         </li>
-        <li ng-if="$ctrl.isSuperAdmin">
-          <a class="protractor-test-admin-link" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/admin" target="_blank" rel="noopener">
+        <li class="nav-item" ng-if="$ctrl.isSuperAdmin">
+          <a class="dropdown-item nav-link protractor-test-admin-link" ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/admin" \
```

Comment:
Do not end lines with a backslash. I don't think you need one in HTML anyway.

Reply:
Updated was not needed PTAL Thanks!